### DISCUSSION TITLE
Missing shim for tail

### DISCUSSION THREAD

**Discussion by eggbean** [ScoopInstaller/Scoop#5770] [Help and Questions] [answered]
I have three scoop packages installed which should include `tail` - `busybox`, `coreutils` and `uutils-coreutils`, but there is no shim for `tail` (there is one for `head`).

I have `%SCOOP%` set to `C:\ProgramData\scoop` so it can be shared between user accounts, if that makes any difference.

Why isn't there a shim for `tail` and how can I fix this?

**Comment by eggbean** [comment]
I have found that I have the same issue for the `tail` command while those same three packages should provide it.

**Comment by r15ch13** [comment, marked as answer]
What is the output of `scoop reset busybox` (or the others)?

**Comment by eggbean** [comment]
The output was too long to easily copy. I didn't think of using `tee` before doing it but, yes, that did seem to fix the issue. Thanks.

I think this may have happened due to using an elevated account to do stuff in the shared directory? I've had to `git config --global --add safe.directory C:/ProgramData/scoop` do stop warning regarding this.
